src/pages/cli/graphql/custom-business-logic.mdx

```diff
@@ -548,6 +548,90 @@ export class cdkStack extends cdk.Stack {
   }
 }
 ```
+#### Pipeline Resolver
+
+```ts 
+import * as cdk from '@aws-cdk/core';
+import * as AmplifyHelpers from '@aws-amplify/cli-extensibility-helper';
+import * as appsync from '@aws-cdk/aws-appsync';
+import { AmplifyDependentResourcesAttributes } from '../../types/amplify-dependent-resources-ref';
+
+const beforeMappingVTL =`
+<YOUR CUSTOM VTL REQUEST MAPPING TEMPLATE HERE>
+`
+const afterMappingVTL =`
+<YOUR CUSTOM VTL RESPONSE MAPPING TEMPLATE HERE>
+`
+const function1requestVTL=`
+<YOUR CUSTOM VTL FUNCTION 1 REQUEST MAPPING TEMPLATE HERE>
+`
+const function1responseVTL=`
+<YOUR CUSTOM VTL FUNCTION 1 RESPONSE MAPPING TEMPLATE HERE>
+`
+const function2requestVTL=`
+<YOUR CUSTOM VTL FUNCTION 2 REQUEST MAPPING TEMPLATE HERE>
+`
+const function2responseVTL=`
+<YOUR CUSTOM VTL FUNCTION 2 RESPONSE MAPPING TEMPLATE HERE>
+`
+export class cdkStack extends cdk.Stack {
+  constructor(scope: cdk.Construct, id: string, props?: cdk.StackProps, amplifyResourceProps?: AmplifyHelpers.AmplifyResourceProps) {
+    super(scope, id, props);
+    /* Do not remove - Amplify CLI automatically injects the current deployment environment in this input parameter */
+    new cdk.CfnParameter(this, 'env', {
+      type: 'String',
+      description: 'Current Amplify CLI env name',
+    });
+    
+    // Access other Amplify Resources 
+    const retVal:AmplifyDependentResourcesAttributes = AmplifyHelpers.addResourceDependency(this, 
+      amplifyResourceProps.category, 
+      amplifyResourceProps.resourceName, 
+      [{
+        category: "api",
+        resourceName: "<YOUR-API-NAME>"
+      }]
+    );
+   
+    
+   const function1 = new appsync.CfnFunctionConfiguration(this,"function1",{
+    apiId: cdk.Fn.ref(retVal.api.replaceWithAPIName.GraphQLAPIIdOutput),
+    dataSourceName: "NONE_DS", // DataSource name
+    functionVersion: "2018-05-29",
+    name: "function1",
+    requestMappingTemplate: function1requestVTL,
+    responseMappingTemplate: function1responseVTL
+   })
+   const function2 = new appsync.CfnFunctionConfiguration(this,"function2",{
+    apiId: cdk.Fn.ref(retVal.api.replaceWithAPIName.GraphQLAPIIdOutput),
+    dataSourceName: "TodoTable",  // DataSource name
+    functionVersion: "2018-05-29",
+    name: "function2",
+    requestMappingTemplate: function2requestVTL,
+    responseMappingTemplate: function2responseVTL
+   })
+
+  const resolver = new appsync.CfnResolver(this, "pipeline-resolver", {
+    // apiId: retVal.api.new.GraphQLAPIIdOutput,
+    // https://github.com/aws-amplify/amplify-cli/issues/9391#event-5843293887
```

Comment:
could you remove these comments and utilize a callout if you would like to add any warnings.